events/loresmt2022.md

```diff
@@ -32,7 +32,7 @@ Github: [github.com/loresmt](https://github.com/loresmt)
 
 |     |     |
 | --- | --- |
-| Papers deadline | 18 July |
+| Submission Open Until| July 30, 2022 (Saturday) at 23:59 (Anywhere on Earth) |
```

Comment:
To follow our [Style Guide](http://machinetranslate.org/style) and the structure in Events, I would suggest the following:

```
|     |     |
| --- | --- |
| Submission deadline | 30 July |
| Notification of acceptance | 22 August |
| Camera-ready papers deadline | 5 September |
| LoResMT workshop | 16 October |

* All deadlines are anywhere on Earth.
```

I deleted the year, as it is implied.

Reply:
No problem. Please make change another thing: "type: Online"